geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -1,4 +1,4 @@
-<% set_title("Redis API for", product_name) %>
+<% set_title(product_name, "API for Compatibility with Redis") %>
```

Comment:
set_title(product_name, "APIs compatible with Redis")

Note we should update the file names to reflect the new naming structure...
geode_apis_compatible_with_redis.html. (we'll also need to update links in other files).